### PR TITLE
Change prometheus volume type to host path so that the data can be

### PR DESCRIPTION
**Pull request by davidewatson**
archived after the pod is deleted (KRAK-346).

### CODE REVIEW

File: prometheus/prometheus.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ spec:
       restartPolicy: Always
       volumes:
       - name: data
-        emptyDir: {}
+        hostPath: /promdash
```
Comment (spiffxp) [inline prometheus/prometheus.yaml]: should this be `prometheus` instead?  also worth considering this is going to go on the root volume, I'm not sure whether emptyDir volumes were getting put on the larger EBS mount or not

Comment (spiffxp) [inline prometheus/prometheus.yaml]: OK, so this has some dependence upon a properly configured volume mounted to 'promdash' existing on the node this is deployed to.  I'd feel more comfortable if this rendered as a template instead, so kraken gets to control what the volume is called at `kubectl create -f` time

File: prometheus/prometheus.yaml [hunk 2]
```diff
@@ -83,6 +83,9 @@ spec:
         kraken-node: node-001
       restartPolicy: Always
       volumes:
+      - name: prometheus
+        hostPath:
+          path: /prometheus
```
Comment (spiffxp) [inline prometheus/prometheus.yaml]: something like replacing this with `{{services.prometheus_volume_host_path}}` or a better name... we do something similar for passing ip's of components to prometheus

**Comment by davidewatson** [comment]
OBE due to k2.

**Pull request closed without merging**